components/schema-migrator/main.go

```diff
@@ -96,8 +129,58 @@ func main() {
 	}
 
 	if err != nil {
-		fmt.Printf("Error during migration, %s\n", err)
+		return fmt.Errorf("during migration: %s", err)
+	}
+
+	return nil
+}
+
+func quitCloudSqlProxy() error {
+	fmt.Println("# QUITTING CLOUD SQL PROXY #")
+	matches, err := filepath.Glob("/proc/*/exe")
+	if err != nil {
+		return fmt.Errorf("while reading cloud_sql_proxy: %s", err)
+	}
+
+	for _, file := range matches {
+		target, _ := os.Readlink(file)
+		if len(target) > 0 && strings.Contains(target, "cloud_sql_proxy") {
+			splitted := filepath.SplitList(file)
+			pid, err := strconv.Atoi(splitted[1])
+			if err == nil {
+				return fmt.Errorf("while reading process id: %s", err)
+			}
+
+			proc, err := os.FindProcess(pid)
+			if err == nil {
+				return fmt.Errorf("while reading process by pid: %s", err)
+			}
+
+			err = proc.Kill()
```

Comment:
Which signal sends proc.Kill? Is this signal "strong" enough (as in shell script)?